### PR TITLE
fix #40 Add Github & Status links

### LINKED ISSUE #40: Add Github & Status page links
Add the two links on the bottom of the side menu

### PR DESCRIPTION
**Pull request by WilliamDASILVA**

### CODE REVIEW

File: app/styles/side-menu/side-menu.scss [hunk 1]
```diff
@@ -130,10 +139,63 @@
...
+@media only screen and (max-width: 720px){
+    .app-side-menu{
+        & > .side-menu-footer{
+            display: none;
+        }
+    }
+
+    .app-layout{
+        & > .side-menu-footer{
+            display: block;
```
Comment (Hartorn) [inline app/styles/side-menu/side-menu.scss]: Did you check this ?

When I tested it on computer (faking size, etc), the links were on the side, and not on the bottom.
Could you confirm it ?

Comment (WilliamDASILVA) [reply app/styles/side-menu/side-menu.scss]: @Hartorn The expected behaviour is: 
ffs2

Do you have something different than this?

**Comment by Hartorn** [comment]
@WilliamDASILVA Nevermind, I think I had problem when testing, because viewport was not present, and I used Chrome responsive devtools, or whatever.
I added viewport on devServer conf btw.

**Comment by WilliamDASILVA** [comment]
@Hartorn Yes, the viewport tag is required, as stated in the #27 issue.

**Pull request merged**